### DISCUSSION TITLE
Seeking Advice on Quickly Obtaining Sulci Labels

### DISCUSSION THREAD

**Discussion by HusBunia** [brainvisa/brainvisa.github.io#131] [Q&A] [answered]
Hi everyone,

I'm wondering if there's a way to quickly obtain sulci labels, similar to the process described in morpho-deepsulci. Unfortunately, I've discovered that the deepsulci Docker image is currently not functioning correctly. Is it possible to achieve this by importing certain Python libraries?

I greatly appreciate any assistance you can provide!

**Comment by ylep** [comment, marked as answer]
The Deep-CNN-based sulci labelling is now part of BrainVISA since release 5.0.0, you can use the latest release of BrainVISA as available on https://brainvisa.info/ . Well, actually there is a known issue with sulci labelling in version 5.1.0 (https://github.com/brainvisa/aims-free/issues/96) and we will release BrainVISA 5.1.1 in a few hours, so you may want to wait just a bit.

The [morpho-deepsulci-docker](https://github.com/LeonieBorne/morpho-deepsulci-docker) Docker image is not maintained anymore, we will remove the link to that project (thanks for letting us know that it is still referred to on the morpho-deepsulci readme).
